src/test/java/com/taskadapter/redmineapi/IssueManagerIT.java

```diff
@@ -759,6 +759,11 @@ public void issueTargetVersionIsSetWhenCreatingOrUpdatingIssues() throws Excepti
         issueManager.update(createdIssue);
         Issue updatedIssue = issueManager.getIssueById(createdIssue.getId());
         assertThat(updatedIssue.getTargetVersion().getName()).isEqualTo(version2Name);
+        
+        createdIssue.setTargetVersion(null);
```

Comment:
this looks like a logically separate test given that the test name is "issueTargetVersionIsSetWhenCreatingOrUpdatingIssues".  this new code is checking that target version can be erased